src/main/java/it/smartcommunitylab/aac/webauthn/persistence/WebAuthnUserAccount.java

```diff
@@ -0,0 +1,135 @@
+package it.smartcommunitylab.aac.webauthn.persistence;
+
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotNull;
+
+import com.yubico.webauthn.data.ByteArray;
+
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import it.smartcommunitylab.aac.SystemKeys;
+import it.smartcommunitylab.aac.core.model.UserAccount;
+
+@Entity
+@Table(name = "webauthn_users", uniqueConstraints = @UniqueConstraint(columnNames = { "provider_id", "username" }))
+@EntityListeners(AuditingEntityListener.class)
+public class WebAuthnUserAccount implements UserAccount {
+
+    private static final long serialVersionUID = SystemKeys.AAC_CORE_SERIAL_VERSION;
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    // TODO: civts, use converter
+    @Column(unique = true)
+    private String userHandle;
+    @OneToMany(mappedBy = "parentAccount", fetch = FetchType.LAZY, orphanRemoval = true)
+    private Set<WebAuthnCredential> credentials;
```

Comment:
remove reference

Reply:
Addressed in ef11f270f4c510ec299cbf24206bf3a88ba45c4d